src/main/resources/db/changelog/changes/03-create-roles-table.yaml

```
@@ -0,0 +1,24 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: VdBondarev
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: VARCHAR(255)
+                  constraints:
+                    unique: true
+                    nullable: false
+      rollback:
+        - dropTable:
+            tableName: roles
```

```suggestion
                    nullable: false
              - column:
                  name: name
                  type: VARCHAR(255)
                  constraints:
                    unique: true
                    nullable: false
      rollback:
        - dropTable:
            tableName: roles
            
```
don't forget one empty line at the EOF

---

can you explain to me why we need an empty line at the end of file? Or is it just that stuff you always should do without explanation?